src/odemis/driver/simulated.py

```
@@ -462,3 +466,507 @@ def __init__(self, name, role, axes, ranges=None, choices=None, **kwargs):
+
+        return rays
+
+    def _hit_lens(
+        self,
+        vector_in,
+        pos_in,
+        n1=1,
+        n2=1.458461,
+        lens_diam=50,
+        focal_length=200,
+        lens_center=[0, 2.8],
```

Ruff (0.14.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)